src/main/java/controller/PassengersController.java

```diff
@@ -0,0 +1,124 @@
+        }
+    }
+
+    public void create(String name,
+                       String email,
+                       String phoneNumber,
+                       int age,
+                       int startLocationId,
+                       int stopLocationId,
+                       int busId) {
+        Statement statement = databaseHandler.createStatement();
+
+        if (statement != null) {
+            try {
+                statement.executeUpdate("INSERT INTO passengers " +
+                        "(fullName," +
+                        " email," +
+                        " phoneNumber," +
+                        " age," +
+                        " startLocationId," +
+                        " stopLocationId," +
+                        " busId)" +
+                        " VALUES" +
+                        "('" + name +
+                        "', '" + email +
+                        "', '" + phoneNumber +
+                        "', '" + age +
+                        "', '" + startLocationId +
+                        "', '" + stopLocationId +
+                        "', '" + busId + "')");
+                statement.close();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    /*
+    create method in all classes to find results from each table using available column name.
+    (Example: using bus number; we can find a bus in mysql database by creating method
+    findByBusNumber(String busNumber) that accepts a string
+     which is the bus number 20A, 42B,
+     etc our function will take that bus number
+     and search mysql database using jdbc for the bus and show the results.)
+     */
+    public void findByPassengerID(int id) {
+        Statement statement = databaseHandler.createStatement();
+        try {
+            ResultSet resultSetPassengers = statement.executeQuery("SELECT * FROM passengers WHERE id = " + id +";");
+            System.out.println("=========================================");
+            System.out.println("Passenger by id # " + id + "information from 'passengers' table");
+
+            while (resultSetPassengers.next()) {
+
+                int myPassengerId = resultSetPassengers.getInt("id");
+                String myPassengerFullName = resultSetPassengers.getString("fullName");
+                String myPassengerEmail = resultSetPassengers.getString("email");
+                String myPassengerPhoneNumber = resultSetPassengers.getString("phoneNumber");
+                int myPassengerAge = resultSetPassengers.getInt("age");
+                int myPassengerStartLocationId = resultSetPassengers.getInt("startLocationId");
+                int myPassengerStopLocationId = resultSetPassengers.getInt("stopLocationId");
+                int myPassengerBusID = resultSetPassengers.getInt("busId");
+                Timestamp timestamp = resultSetPassengers.getTimestamp("createdOn");
+
+                if (myPassengerId == id) {
```

Comment:
This `if (myPassengerId == id)` statement does not work here. It does not check anything. Instead change `while` to `if` in the beginning.

Reply:
done